### DISCUSSION TITLE
我在美团实习的经历与感想 | XX’s Blog

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan.github.io#15] [Announcements]
https://xxchan.github.io/cs/2020/09/02/intern-at-meituan.html 

 Disclaimer：本文提及的内容有效范围仅限于后端业务开发岗位，甚至仅限于我所在部门，仅为个人体验与看法，请酌情参考。

**Comment by xxchan** [comment]
看到一篇关于“分工”的不错的文章：https://erikbern.com/2021/07/23/what-is-the-right-level-of-specialization.html

**Comment by q2333gh** [comment]
Thanks for your blog, it really helps!
